packages/odata-v4/package.json

```diff
@@ -47,6 +47,7 @@
   },
   "devDependencies": {
     "@sap-cloud-sdk/test-services-odata-v4": "^2.10.0",
+    "@sap-cloud-sdk/test-services-e2e": "^2.10.0",
```

Comment:
[req] I guess we have `test-services-odata-v4` and `@sap-cloud-sdk/test-services-e2e` for unit tests and e2e tests respectively.
- I would adjust/use the `test-services-odata-v4`, for the unit test `bound-function-import-request-builder.spec.ts`
- I would not use `@sap-cloud-sdk/test-services-e2e` for unit tests because the spec/client of the e2e tests should be updated after the cds data model updates, which is kind of passive, while our `test-services-odata-v4` is fully under our control.